### PR TITLE
Remove innerloop flow jobs that were missed

### PR DESCRIPTION
**Pull request by jashook**

### CODE REVIEW

**Comment by jashook** [comment]
@dotnet-bot test ci please

File: netci.groovy [hunk 1]
```diff
@@ -3572,6 +3572,15 @@ def static shouldGenerateFlowJob(def scenario, def isPR, def architecture, def c
         }
     }
 
+    // Disable flow jobs for innerloop pr.
+    //
+    // The only exception is windows arm(64)
+    if (isInnerloopTestScenario(scenario) && isPR && os != 'Windows_NT') {
```
Comment (echesakovMSFT) [inline netci.groovy]: Since it's only Windows arm(64) should we check for the right `architecture` here?

**Comment by BruceForstall** [comment]
Don't you want to also remove `Windows_NT x86 Checked Innerloop Build and Test (Jit - TieredCompilation=0` and `Windows_NT x64 Checked Innerloop Build and Test (Jit - TieredCompilation=0`?

**Comment by jashook** [comment]
Thanks, addressed.

**Comment by jashook** [comment]
@dotnet-bot test ci please

File: netci.groovy (hunk 1 above)
Comment (jashook) [reply netci.groovy]: There will be an assert for windows arm and arm64 in 20 lines if it is windows.

**Comment by jashook** [comment]
@dotnet-bot test ci please

File: netci.groovy [hunk 2]
```diff
@@ -2558,6 +2558,12 @@ def static shouldGenerateJob(def scenario, def isPR, def architecture, def confi
         return false
     }
 
+    if (isInnerloopTestScenario(scenario) && isPR) {
```
Comment (BruceForstall) [inline netci.groovy]: I think you should move this to the top and have:

```
    // Innerloop jobs (except corefx_innerloop) are no longer created in Jenkins
    if (isInnerloopTestScenario(scenario)) {
        assert scenario != 'corefx_innerloop'
        return false;
    }

    if (!isPR) {
        if (scenario == 'corefx_innerloop') {
            return false
        }
    }
```

File: netci.groovy (hunk 1 above)
Comment (BruceForstall) [reply netci.groovy]: Can you explain why Windows_NT is exempt?

Comment (jashook) [reply netci.groovy]: The windows arm and arm64 jobs will remain, as we currently do not have a mechanism similar to the whitelist that would keep these machines from being overloaded on Azure Dev Ops

**Comment by jashook** [comment]
@BruceForstall @echesakovMSFT ptal

**Comment by jashook** [comment]
@dotnet-bot test ci please

**Comment by jashook** [comment]
@dotnet-bot test ci please

**Pull request merged**